primus.js

```diff
@@ -621,15 +621,29 @@ Primus.prototype.initialise = function initialise(options) {
     }
   }
 
+  primus.addGlobalListeners();
+
+  return primus;
+};
+
+Primus.prototype.addGlobalListeners = function () {
```

Comment:
Why is there a need for a separate function here? Can't the assignment just stay in the init method?